packages/flutter/test/material/dialog_test.dart

```diff
@@ -4,6 +4,9 @@
 
```

Comment:
you also need a test of what happens if a render object is both a semantics boundary and has the new flag set.